File: SDKs/Aspose.Cells_Cloud_SDK_For_CSharp/src/Com/Aspose/cells/Model/Cells.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Cells.Model {
  public class Cells {
    public int? MaxRow { get; set; }

    public int? MaxColumn { get; set; }

    public int? CellCount { get; set; }

    public LinkElement Rows { get; set; }

    public LinkElement Columns { get; set; }

    public List<LinkElement> CellList { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Cells {\n");
      sb.Append("  MaxRow: ").Append(MaxRow).Append("\n");
      sb.Append("  MaxColumn: ").Append(MaxColumn).Append("\n");
      sb.Append("  CellCount: ").Append(CellCount).Append("\n");
      sb.Append("  Rows: ").Append(Rows).Append("\n");
      sb.Append("  Columns: ").Append(Columns).Append("\n");
      sb.Append("  CellList: ").Append(CellList).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Cells-Cloud-SDK-For-Ruby/lib/aspose_cells_cloud/models/cells.rb
module AsposeCellsCloud
  # 
  class Cells < BaseObject
    attr_accessor :max_row, :max_column, :cell_count, :rows, :columns, :cell_list, :link
    # attribute mapping from ruby-style variable name to JSON key
    def self.attribute_map
      {
        
        # 
        :'max_row' => :'MaxRow',
        
        # 
        :'max_column' => :'MaxColumn',
        
        # 
        :'cell_count' => :'CellCount',
        
        # 
        :'rows' => :'Rows',
        
        # 
        :'columns' => :'Columns',
        
        # 
        :'cell_list' => :'CellList',
        
        # 
        :'link' => :'link'
        
      }
    end

    # attribute type
    def self.swagger_types
      {
        :'max_row' => :'Integer',
        :'max_column' => :'Integer',
        :'cell_count' => :'Integer',
        :'rows' => :'LinkElement',
        :'columns' => :'LinkElement',
        :'cell_list' => :'Array<LinkElement>',
        :'link' => :'Link'
        
      }
    end

    def initialize(attributes = {})
      return if !attributes.is_a?(Hash) || attributes.empty?

      # convert string to symbol for hash key
      attributes = attributes.inject({}){|memo,(k,v)| memo[k.to_sym] = v; memo}

      
      if attributes[:'MaxRow']
        self.max_row = attributes[:'MaxRow']
      end
      
      if attributes[:'MaxColumn']
        self.max_column = attributes[:'MaxColumn']
      end
      
      if attributes[:'CellCount']
        self.cell_count = attributes[:'CellCount']
      end
      
      if attributes[:'Rows']
        self.rows = attributes[:'Rows']
      end
      
      if attributes[:'Columns']
        self.columns = attributes[:'Columns']
      end
      
      if attributes[:'CellList']
        if (value = attributes[:'CellList']).is_a?(Array)
          self.cell_list = value
        end
      end
      
      if attributes[:'link']
        self.link = attributes[:'link']
      end
      
    end

  end
end
